inscriptions/ISic030315.xml

```diff
@@ -0,0 +1,221 @@
+                </respStmt>
+                <respStmt>
+                    <name xml:id="system">system</name>
+                    <resp>automated or batch processes</resp>
+                </respStmt>
+                <respStmt>
+                    <name xml:id="SS" ref="https://orcid.org/0000-0003-3914-9569">Simona Stoyanova</name>
+                    <resp>standardisation of template and tidying up encoding</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>I.Sicily</authority>
+                <idno type="filename">ISic030315</idno>
+                <idno type="TM"/>
+                <idno type="EDR"/>
+                <idno type="EDH"/>
+                <idno type="EDCS"/>
+                <idno type="PHI"></idno>
+                <idno type="URI">http://sicily.classics.ox.ac.uk/inscription/ISic030315</idno>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by/4.0/">Licensed under a Creative Commons-Attribution 4.0 licence.</licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc>
+                    <msIdentifier>
+                        <country>Italy</country>
+                        <region>Sicily</region>
+                        <settlement>Santa Croce Camerina</settlement> 
+                        <repository role="museum" ref="http://sicily.classics.ox.ac.uk/museum/093">Museo Archeologico Regionale di Kamarina</repository>
+                        <idno type="inventory">777-778</idno><!-- check these inv.nos. -->
+                        <altIdentifier>
+                            <settlement/>
+                            <repository/>
+                            <idno type="old"/>
+                        </altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <textLang mainLang="grc">Ancient Greek</textLang>    
+                    </msContents>
+                    <physDesc>
+                        <objectDesc>
+                            <supportDesc>
+                                <support>
+                                    <p>Tessera.</p>
+                                    <material ana="#material.metal.lead" ref="https://www.eagle-network.eu/voc/material/lod/108.html">lead</material>
+                                    <objectType ana="#object.tessera" ref="http://www.eagle-network.eu/voc/objtyp/lod/276.html">tessera</objectType>
+                                    <dimensions>
+                                        <height unit="cm">2.8-2.5</height><!-- two contiguous fragments: h1= 2.8, h2= 2.5 -->
+                                        <width unit="cm">1.8-1.5</width><!-- two contiguous fragments: w1= 1.8, w2= 1.5 -->
+                                        <width unit="cm">6</width><!-- two contiguous fragments: total lenght 6 cm -->
```

Comment:
This will either need msParts, or simply description; the main measurement if just describing should be the total one.